docs/30-production/25-secrets.md

```diff
@@ -44,6 +44,39 @@ acorn run -s my-predefined-creds:user-creds registry.example.com/myorg/image
 
 When this Acorn runs it will use the values in the `my-predefined-creds` secret.
 
+### Modifying a Secret
+
+#### Editing a secret
+
+Once you have created a secret if you need to edit it you can use the `acorn secret edit` command. This will allow you to interactively modify the secret.
+
+```shell
+acorn secret edit my-app-secret-creds
+```
+
+This will open up your editor and you can modify the values. When you save and exit the secret will be updated. Changing this value will cause containers to be restarted unless the Acorn using them is configured to ignore updates.
```

Comment:
Making this the same as below
```suggestion
This will open up your editor and you can modify the values. When you save and exit the secret will be updated. Changing the secret values will cause containers consuming the secret to be restarted. This behavior can be changed if the Acorn is configured to ignore updates by the author.
```